contracts/main/proxy-actions/FathomStablecoinProxyActions.sol

```diff
@@ -349,6 +349,9 @@ contract FathomStablecoinProxyActions {
         uint256 _stablecoinAmount, // [wad]
         bytes calldata _data
     ) public payable {
+        //Minimum borrowing requirement
+        require(IManager(_manager).minimumDebtCheck(_stablecoinAmount), "ProxyActions/lockXDCAndDraw-debtTooLow");
```

Comment:
about minimum borrow. I would consider moving checking logic from proxy action contract to PositionManager itself. And maybe it makes sence to add a similar check to other functions like adjustPosition

Reply:
okei, ser. will look into adjustPosition.